src/style.css

```diff
@@ -0,0 +1,81 @@
+:root {
+  --star: url(images/star.svg);
+  --star-active: url(images/star-active.svg);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars-block {
+  display: flex;
+  padding-top: 2px;
+}
+
+.stars-block > * {
+  height: 13px;
+  width: 13px;
+  background-repeat: no-repeat;
+  margin: 0 3px;
+
+}
+
+.stars-block > :first-child {
+  margin-left: 1px;
+}
+
+.stars--0 > * {
+  background-image: var(--star);
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--1 :nth-child(2),
+.stars--1 :nth-child(3),
+.stars--1 :nth-child(4),
+.stars--1 :nth-child(5) {
+  background-image: var(--star);
+}
```

Comment:
there is one cool way how to write this, its looks the same but much shorter (it takes only one line) and nth-child contains (n) letter